DotaStats/Services/ImageService.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+import UIKit
+
+protocol ImageService: AnyObject {
+    func loadWithUrl(_ url: String, _ completion: @escaping (Result<UIImage, HTTPError>) -> Void) -> Cancellable?
+}
+
+final class ImageServiceImp: ImageService {
+    private let urlSession: URLSession = {
+        let config = URLSessionConfiguration.default
+        return URLSession(configuration: config)
+    }()
```

Comment:
Кажется, что это можно сократить до 
```suggestion
    private let urlSession: URLSession = .init(configuration: URLSessionConfiguration.default)
```